UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/Common.js

```diff
@@ -0,0 +1,73 @@
+﻿document.addEventListener("DOMContentLoaded", function () {
+    HashLinkTab();
```

Comment:
As mentioned above, please move all relevant changes to site.js and remove common.js file.